WordPress/WordPressUITests/Tests/MainNavigationTests.swift

```diff
@@ -17,26 +17,41 @@ class MainNavigationTests: XCTestCase {
         removeApp()
     }
 
-    func testTabBarNavigation() throws {
+    // We run into an issue where the People screen would crash short after loading.
+    // See https://github.com/wordpress-mobile/WordPress-iOS/issues/20112.
+    //
+    // It would be wise to add similar tests for each item in the menu (then remove this comment).
+    func testLoadsPeopleScreen() throws {
         XCTAssert(MySiteScreen.isLoaded(), "MySitesScreen screen isn't loaded.")
 
-        _ = try mySiteScreen
-            .tabBar.goToReaderScreen()
+        mySiteScreen.goToMenu()
+        mySiteScreen.goToPeople()
 
-        XCTAssert(ReaderScreen.isLoaded(), "Reader screen isn't loaded.")
+        // We don't have a ScreenObject for the people screen,
+        // so we just check that the previous screen is not loaded
+        XCTAssertFalse(MySiteScreen.isLoaded(), "MySitesScreen screen isn't loaded.")
```

Comment:
I think this would fail on iPad, since the "My Site" page is still visible? Maybe we can assert the tabs("Users", "Folowers", etc) on the People screen exist instead? Or adding an accessibility identifier to the view controller, so that we can find it in the view hierarchy in the UI test and assert it's loaded?

Reply:
[The iPad test](https://buildkite.com/automattic/wordpress-ios/builds/12326#018639d6-9e67-4921-a128-e668aabebe04/597-4248) indeed failed.

Reply:
Oh, interesting... 🤔 I'll have to look into that.
 
The iPad failure and the lack of failure I experienced might be at two different levels. On iPhone, the people screen loads but doesn't crash. I haven't checked the iPad yet, but it's likely due to what you suggest, hence we need better assertsions.